src/tools/pbs_python.c

```diff
@@ -2310,11 +2310,37 @@ main(int argc, char *argv[], char *envp[])
 		char *largv[3];
 		int ll;
 		char *pc, *pc2;
+
+#ifdef SYSTEM_PYTHON_PATH
+		snprintf(python_path, MAXPATHLEN, "%s", SYSTEM_PYTHON_PATH);
+		pc = strdup(SYSTEM_PYTHON_PATH);
+		if(pc == NULL) {
+			fprintf(stderr, "Out of memory\n");
+			return 1;
+		}
+		pc2 = strstr(pc,"bin/python");
+		if(pc2 == NULL) {
+			fprintf(stderr, "Python executable not found!\n");
+			return 1;
+		}
+		ll = pc2 - pc;
+		if(ll <= 0) {
+			fprintf(stderr, "Python home not found!\n");
+			return 1;
+		}
+		snprintf(python_prefix, ll, "%s", SYSTEM_PYTHON_PATH);
```

Comment:
Hi @bhroam,
       I have initialized the python_prefix string with '\0' as per your comment. Please have 
a look at the changes.
Thanks,
Rampranesh M